aspnetcore/blazor/common/samples/5.x/BlazorServerEFCoreSample/BlazorServerDbContextExample/Pages/Index.razor

```diff
@@ -0,0 +1,242 @@
+﻿@page "/"
+@page "/{Page:int}"
+
+@using Microsoft.EntityFrameworkCore
+
+@inject IContactFilters Filters
+@inject IDbContextFactory<ContactContext> DbFactory
+@inject NavigationManager Nav
+@inject GridQueryAdapter QueryAdapter
+
+<h1>Contacts</h1>
+<GridWrapper @ref="Wrapper"
+             FilterChanged="async () => await ReloadAsync()"
+             DeleteRequested="id => Wrapper.DeleteRequestId = id">
+    <div class="container-fluid contacts-grid">
+        <div class="row">
+            <div class="col-4"><NameToggle /></div>
+            <div class="col-8"><TextFilter /></div>
+        </div>
+        <div class="row">&nbsp;</div>
+        <div class="row">
+            <div class="col-6">
+                Page @Filters.PageHelper.Page of @Filters.PageHelper.PageCount:
+                showing @Filters.PageHelper.PageItems of
+                @Filters.PageHelper.TotalItemCount items.
+                <a disabled="@(Filters.Loading || !Filters.PageHelper.HasPrev)"
+                   class="btn btn-primary @IsDisabled(Filters.PageHelper.HasPrev)"
+                   href="/@Filters.PageHelper.PrevPage">
```

Comment:
Why the leading slash on the URLs like this? Does it work OK without that? I expect so since it's resolving relative to the base href. If so it's better not to put on the leading slash, as that would break any case where this was deployed to a non-root URL.

Reply:
I changed these to avoid leading slash and tested - appears to work fine.